packages/javascript-decorators/src/tests/memoize-server.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment node
+ */
+
+import memoize from '../memoize';
+
+describe('memoize()', () => {
+  describe('server', () => {
+    const addOne = (number: number) => number + 1;
```

Comment:
I would not share these between tests personally, it is very confusing in the context of testing number of calls